Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.cxx

```diff
@@ -295,8 +296,14 @@ void vtkMRMLMarkupsDisplayNode::PrintSelf(ostream& os, vtkIndent indent)
   vtkMRMLPrintBooleanMacro(SliceProjectionOutlinedBehindSlicePlane);
   vtkMRMLPrintVectorMacro(SliceProjectionColor, double, 3);
   vtkMRMLPrintFloatMacro(SliceProjectionOpacity);
-  vtkMRMLPrintFloatMacro(ActiveComponentType);
-  vtkMRMLPrintFloatMacro(ActiveComponentIndex);
+  {
```

Comment:
What about adding a `vtkMRMLPrintMapMacro`

Reply:
I could create one, but it wouldn't be usable in this case due to the custom structure it has as values
(os << it->first << ": " << it->second.Type << ", " << it->second.Index;)

Reply:
Good point, it is not worth the indirection then.